src/Microsoft.Extensions.Logging.Filter/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": "1.0.0-*",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/aspnet/logging"
+  },
+  "description": "Logging filters.",
```

Comment:
And add `tags` too. I think just `logging` is enough (that's the common one all the logging-related projects have).